test/controllers/graphql_controller_3_test.rb

```diff
@@ -61,7 +61,7 @@ def setup
     end
     create_project_media project: p, user: u, disable_es_callbacks: false
     pm = create_project_media project: p, disable_es_callbacks: false
-    pm.archived = true
+    pm.archived = 1
```

Comment:
Why not use the constants in the test too?

Reply:
Fixed by using constants